lib/matplotlib/axes/_axes.py

```diff
@@ -5624,7 +5624,7 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
         return im
 
     @staticmethod
-    def _pcolorargs(funcname, *args, allmatch=False):
+    def _pcolorargs(funcname, *args, shading='flat'):
```

Comment:
Looks like the comment below needs to be updated to remove the reference to allmatch.

Reply:
Changed comment to read:

```python
        # - create X and Y if not present;
        # - reshape X and Y as needed if they are 1-D;
        # - check for proper sizes based on `shading` kwarg;
        # - reset shading if shading='auto' to flat or nearest
        #   depending on size;
```